elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -12,28 +12,26 @@ $(function() {
 
     <div class="row">
         <div class="column {{dropzone_layout}}">
-            <ul class="list-group-item list-group-item-primary header">
-                <li><strong>{{source-header}}</strong>
-                    <a role="button" class="btn btn-link btn-sm disabled" data-toggle="popover">
-                        <i class="fa fa-question-circle" aria-hidden="true" style="visibility: hidden;"></i>
-                    </a>
-                </li>
-            </ul>
-            <ul id="order-blocks-options-{{uuid}}" name="{{answer_name}}" class="pl-order-blocks-connected-sortable {{inline}}">
+            <div class="card-header list-group-item-primary">
+                {{source-header}}
+                <a style="visibility:hidden;" role="button" class="btn btn-light btn-sm" data-toggle="popover" data-html="true" title="Order Blocks" data-content="{{help_text}}" data-trigger="focus" tabindex="0">
+                    <i class="fa fa-question-circle" aria-hidden="true"></i>
+                </a>
```

Comment:
Without this hidden button here, the headers of the source and target block dropzones are different sizes, which looks funny. But I'm sure there is a better way to do this than having a hidden button. Maybe we actually want to have the help button on both the source and target dropzones? When they are arranged vertically, it looks fine, but it looks a little odd when they are arranged horizontally.

Reply:
If you want them to be the same height, why not just set a specific height on the container, e.g. `height: 3rem`?

I agree it would look strange to have the same popover on both panels.